autoload/magit/utils.vim

```diff
@@ -55,7 +60,7 @@ endfunction
 function! magit#utils#system(...)
 	let dir = getcwd()
 	try
-		call magit#utils#lcd(magit#git#top_dir())
+		execute magit#utils#chdir(magit#git#top_dir())
```

Comment:
Can you explain why you changed `call` to `execute`? And why don't you use `execute` everywhere in that case?